index.html

```diff
@@ -1,41 +1,291 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Gergito Portfolio</title>
-    <link rel="stylesheet" type="text/css" href="style.css">
-</head>
+
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="description" content="Gergito Kusie | A software developer as full-stack" />
+    <meta name="keywords" content="HABASM, Gergito Kusie, full-stack developer, app developer" />
+    <meta name="author" content="Gergito Kusie" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/png" sizes="32x32" href="./img/favicon-32x32.png" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Gergito Portofilio</title>
+  </head>
+
 <body>
-    
-    <header>
-        <nav id="top-nav">
-          <a class="logo" href="#">My Logo</a>          
-          <nav><img class="menu" src="img/menu.png" alt="Menu" ></nav>
-        </nav>
+  <div class="main_container">
+    <header class="header background-white">
+      <nav class="navigation_bar flex">
+        <a href="#" class="navigation_bar__link link size-20 weight-700 line_hieght-20">GERGITO</a>
+        <img class="navigation_bar__hamb" src="img/hamburguer.png" alt="hamburguer icon" />
+      </nav>
+    </header>
+
+    <section class="head flex">
+      <h1 class="head__title size-40 weight-700 line_hieght-52">
+        I'm GERGITO <span>Glad to see you!</span>
+      </h1>
+      <p class="head__dsc size-16 weight-400 line_hieght-24">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+      <a href="#" class="head__link link size-16 weight-500 line_hieght-24 clr-p-400">My Projects</a>
+      <ul class="icons_list ul flex">
+        <li class="icons_list__li">
+          <a href="https://twitter.com/gergitok" class="icons_list__link link"><img src="img/twitter.png" alt="twitter icon" /></a>
+        </li>
+        <li class="icons_list__li">
+          <a href="https://www.linkedin.com/in/gergitok/" class="icons_list__link link"><img src="img/Linkedin.svg" alt="linkedin icon" /></a>
+        </li>
+        <li class="icons_list__li">
+          <a href="#" class="icons_list__link link"><img src="img/M.png" alt="M icon" /></a>
+        </li>
+        <li class="icons_list__li">
+          <a href="https://github.com/gergito" class="icons_list__link link"><img src="img/github.svg" alt="github icon" /></a>
+        </li>
+        <li class="icons_list__li">
+          <a href="#" class="icons_list__link link"><img src="img/hand.svg" alt="hand icon" /></a>
+        </li>
+      </ul>
+    </section>
```

Comment:
- [Optional] I noticed you don't have a background image in your headline section as it is on the template. Kindly add a background image as it is on the template. You are required to stick to design as much as possible.
### Figma template
![image](https://user-images.githubusercontent.com/94551484/227542069-9f196a7b-6454-4763-9941-5ab37940a160.png)
### Your design
![image](https://user-images.githubusercontent.com/94551484/227542158-f66df9bf-94ab-4872-b541-d1674a81e092.png)